great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -925,6 +927,9 @@ def _is_bracketed_by_quotes(target: str) -> bool:
         return False
 
 
+SQLAssets = Annotated[Union[TableAsset, QueryAsset], Field(discriminator="type")]
```

Comment:
```suggestion
SQLAssetTypes = Annotated[Union[TableAsset, QueryAsset], Field(discriminator="type")]
```

Reply:
AssetTypes is sufficiently unambiguous to me because of the module name

Reply:
Done